### ISSUE TITLE
Cannot access std:vector<double> output of computeGroundPartials in python

### ISSUE THREAD

**Issue by jessemapel** [USGS-Astrogeology/swigcsm#20]
The vector cannot be iterated over, you cannot check it's length, and you cannot access its elements.

https://gist.github.com/jessemapel/46f4ab9b09c4dffc106a497584d27d81

**Comment by jessemapel** [comment]
When I killed the notebook server I also go this error message

```
swig/python detected a memory leak of type 'std::vector< double > *', no destructor found.
```

**Comment by jessemapel** [comment]
Fixed by #21
